lib/inspec/base_cli.rb

```diff
@@ -118,6 +118,8 @@ def self.target_options # rubocop:disable Metrics/MethodLength
         desc: "Disable SSL verification on select targets"
       option :target_id, type: :string,
         desc: "Provide a ID which will be included on reports"
+      option :winrm_shell_type, type: :string,
```

Comment:
Is it OK not to have a default? Looking at https://github.com/inspec/inspec/blob/master/lib/inspec/config.rb#L144 I think the default case is that this option won't get passed as a config key through to train?

Reply:
I've updated this now to add "powershell" by default